core/src/main/java/org/mapfish/print/servlet/BaseMapServlet.java

```diff
@@ -18,10 +18,6 @@
  */
 public abstract class BaseMapServlet {
     private static final Logger LOGGER = LoggerFactory.getLogger(BaseMapServlet.class);
-    /**
-     * A logger for logging the print specifications.
-     */
-    protected static final Logger SPEC_LOGGER = LoggerFactory.getLogger(BaseMapServlet.class.getPackage().getName() + ".spec");
```

Comment:
Please grep for `org.mapfish.print.servlet.spec` and adapt the XML files accordingly